app/models/work.rb

```diff
@@ -748,7 +748,10 @@ def chapter_total_display
   # Set the value of word_count to reflect the length of the chapter content
   def set_word_count
     if self.new_record?
-      self.word_count = self.chapters.first.set_word_count
+      self.word_count = 0
```

Comment:
Alas, it turned out that my initial hunch was correct. There appear to be two scenarios - one where the chapters are only stored in memory, and one where they are apparently not in memory and their word count needs to be retrieved from the database. The second occurs when editing works that have been imported: the word counts on those broke when I removed the line that does a database SUM, so I've had to put it back.